### ISSUE TITLE
How do you activate the theme?

### ISSUE THREAD

**Issue by duane** [nashamri/spacemacs-theme#178]
I have it installed, but "load-theme spacemacs" and "load-theme spacemacs-theme" do nothing to load the theme.

**Comment by duane** [comment]
Additionally, "color-theme-select" has no knowledge of the theme. "color-theme-spacemacs-dark" do nothing to activate the theme. It is unclear how you are supposed to activate the theme.

**Comment by nashamri** [comment]
@duane How did you install it? did you use `use-package` or something similar?

**Comment by nickali** [comment]
Think it would help if there were more specific installation instructions.

For example:  "To install the dark theme, put this in your init.el:

`
(use-package spacemacs-theme
  :ensure t)
(load-theme 'spacemacs-dark t)
`

**Comment by haziz** [comment]
I agree, we need better documentation on how to apply or activate the theme once installed.

I installed the theme using the NonGNU archive and still have no idea how to activate it.

Adding 

(use-package spacemacs-theme :ensure t) (load-theme 'spacemacs-dark t)

to my init.el file did NOT work.

Thanks.

**Comment by matheusalpes** [comment]
The following code block worked for me:

```
(use-package spacemacs-theme
  :config
  (require 'spacemacs-theme)
  (deftheme spacemacs-light "Spacemacs theme, light version)
  (create-spacemacs-theme 'light 'spacemacs-light)
  (provide-theme 'spacemacs-light)
  (load-theme 'spacemacs-light t))
```
